Assets/Scripts/MainCharacter/MCAgent.cs

```diff
@@ -86,7 +106,24 @@ private Vector2 PlayerMovement()
 
             outMove = PlayerMoveDirection() * playerSpeed;
 
-            return outMove * Time.fixedDeltaTime;
+            return outMove * Time.deltaTime;
+        }
+
+        public void EnableSpells()
+        {
+            spellsEnabled = true;
+        }
+
+        /// <summary>
+        /// Coroutine to temporarily disable casting spells
+        /// </summary>
+        /// <param name="seconds">Number of seconds to wait</param>
+        /// <returns></returns>
+        private IEnumerator castDelay(float seconds)
```

Comment:
This function might be a problem in the future if we have pausing.   We don't have any problems right now.